service/networking.py

```diff
@@ -1,8 +1,303 @@
-from atmosphere.settings import ATMOSPHERE_PRIVATE_KEYFILE
+"""
+Atmosphere network
+
+This file will help a driver infer the strategy to use when creating or destroying a "project network".
+"""
+import random
+
+from neutronclient.common.exceptions import NeutronClientException, NotFound
```

Comment:
`neutronclient` isn't in our `requirements.txt` it looks like it was remove in 2014?